### PR TITLE
Added StfalconTinymceBundle

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q             | A
| ------------- | ---
| License       | MIT

About the bundle:

* 600,000 downloads
* https://github.com/stfalcon/TinymceBundle

---

@stfalcon you are the creator of this bundle, so it'd be great if you could review this. The idea of the recipes is to provide no configuration or a minimal configuration. The default config options of the bundle should be the good ones for most of the users. So, I propose to include in the recipe config only the two options that you may need to change or at least control more closely.

Do you agree? Would you add more must-have config options? Would you remove these? Thanks!

### CODE REVIEW

**Comment by stfalcon** [comment]
@javiereguiluz yes, of course. thanks dude! ;)

**Comment by fabpot** [comment]
@symfony-flex-server review please

File: stfalcon/tinymce-bundle/2.1/manifest.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "bundles": {
+        "Stfalcon\\Bundle\\TinymceBundle\\StfalconTinymceBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "etc/": "%ETC_DIR%/"
```
Comment (Nyholm) [inline stfalcon/tinymce-bundle/2.1/manifest.json]: This need to be updated. 

```
"config/": "%CONFIG_DIR%/"
```

**Comment by Nyholm** [comment]
Thank you @javiereguiluz. I asked the bundle to tag 2.2. Could you move the recipe from 2.1 to 2.2? After that I think you are done.

**Comment by fre5h** [comment]
Tag `v2.2.0` for this bundle is ready. You can use it.

**Comment by Nyholm** [comment]
@symfony-flex-server review please

**Comment by Nyholm** [comment]
@symfony-flex-server review please

**Comment by javiereguiluz** [comment]
@Nyholm this PR stalled. What should we do here? Update something? Close it? Thanks!

**Comment by Nyholm** [comment]
We are blocked by https://github.com/stfalcon/TinymceBundle/issues/225. And the maintainers do not seam to be eager to reply to me...

@stfalcon or @metalslave, could you spend 3 minutes on https://github.com/stfalcon/TinymceBundle/pull/232 ?

**Comment by javiereguiluz** [comment]
I'm closing this because the upstream bundle is no longer active or it's not interested in this. Thank you and sorry for having wasted your time.

**Pull request closed without merging**